### DISCUSSION TITLE
Submit ideas 💡

### DISCUSSION THREAD

**Discussion by romslf** [romslf/Ultimate-Proxy#4] [Ideas]
First, thank you for thinking of proposing your idea ! 💡

Keep in mind that is a **PROPOSITION**, so I can denie it.

**Comment by miguelcarv1973** [comment]
will be possible to add CFX?

**Reply by romslf** [reply]
Hello,
thank you for your input, I will take a look when I find some time, meanwhile you can join the [discord server](https://discord.gg/Bt4Km5gD) to be notified on updates.

**Reply by romslf** [reply]
> will be possible to add CFX?

Done

**Comment by Canefis** [comment]
Hi,
Please add zil node direct mining
Thanks

**Reply by romslf** [reply]
Hey **@Canefis**,

I responded to you in the issue you opened yesterday.

Best regards,
**@romslf**
